src/util/toast.ts

```diff
@@ -0,0 +1,11 @@
+import { toast as toastifyToast } from 'react-toastify';
+
+/**
+ * Displays a toast which cannot be duplicated and
+ * continues ticking, even when the window loses focus.
+ */
+export const toast = (content: string, options: any = {}) => {
+  if (!toastifyToast.isActive(content)) {
+    return toastifyToast(content, { ...options, ...{ toastId: content } });
+  }
```

Comment:
Possibly return something in case the toast is not being shown again. You never use the return value, but could use it to dismiss the toast or change appearance / content.

Reply:
Also, using the content as a `toastId` is pretty dirty but works. JS has no hashCode or something which could be used here.